java/test/src/main/java/org/ray/api/test/MultiThreadingTest.java

```diff
@@ -42,6 +43,32 @@ public Integer echo(int num) {
     }
   }
 
+  @RayRemote
+  public static class ActorIdTester {
+
+    public ActorIdTester() {
+      Assert.assertNotEquals(Ray.getRuntimeContext().getCurrentActorId(), ActorId.NIL);
+    }
+
+    @RayRemote
+    public ActorId getCurrentActorId(boolean async) {
```

Comment:
`RuntimeContextTest` already tests the main thread case, we can only test the background thread case here.

Reply:
Thanks!